qir/qsharp-target-tests/TargetTestScripts/__init__.py

```diff
@@ -1,63 +1,509 @@
-              "Decompositions/HFromSinglyControlled.qs", "Decompositions/I.qs", "Decompositions/Measure.qs", "Decompositions/MResetXExplicit.qs", "Decompositions/MResetYExplicit.qs", "Decompositions/MResetZExplicit.qs", "Decompositions/MWithPostPrep.qs", "Decompositions/PreparePostMNoop.qs", "Decompositions/RFromSinglyControlledR1.qs", "Decompositions/R1FromSinglyControlled.qs", "Decompositions/R1Frac.qs", "Decompositions/ResetAll.qs", "Decompositions/RFrac.qs", "Decompositions/RxFromSinglyControlled.qs", "Decompositions/RyFromSinglyControlled.qs", "Decompositions/RzFromSinglyControlled.qs", "Decompositions/SFromSinglyControlled.qs", "Decompositions/SWAPFromSinglyControlled.qs", "Decompositions/TFromSinglyControlled.qs", "Decompositions/XFromSinglyControlled.qs", "Decompositions/YFromSinglyControlled.qs", "Decompositions/ZFromSinglyControlled.qs", "Decompositions/SetToBasisState.qs", "Decompositions/Utils.qs"]
+available4 = [
+    "Decompositions/CCNOT.qs",
+    "Decompositions/CCNOTAdjoint.qs",
+    "Decompositions/CCNOTControlled.qs",
+    "Decompositions/CCNOTAdjointControlled.qs",
+    "Decompositions/CCNOTControlledAdjoint.qs",
+    "Decompositions/CNOT.qs",
+    "Decompositions/CNOTAdjoint.qs",
+    "Decompositions/CNOTControlled.qs",
+    "Decompositions/CNOTAdjointControlled.qs",
+    "Decompositions/CNOTControlledAdjoint.qs",
+    "Decompositions/CX.qs",
+    "Decompositions/CXAdjoint.qs",
+    "Decompositions/CXControlled.qs",
+    "Decompositions/CXAdjointControlled.qs",
+    "Decompositions/CXControlledAdjoint.qs",
+    "Decompositions/CY.qs",
+    "Decompositions/CYAdjoint.qs",
+    "Decompositions/CYControlled.qs",
+    "Decompositions/CYAdjointControlled.qs",
+    "Decompositions/CYControlledAdjoint.qs",
+    "Decompositions/CZ.qs",
+    "Decompositions/CZAdjoint.qs",
+    "Decompositions/CZControlled.qs",
+    "Decompositions/CZAdjointControlled.qs",
+    "Decompositions/CZControlledAdjoint.qs",
+    # TODO(tfr): "Decompositions/ExpFrac.qs",
+    # TODO(tfr): "Decompositions/ExpFracAdjoint.qs",
+    # TODO(tfr): "Decompositions/ExpFracControlled.qs",
+    # TODO(tfr): "Decompositions/ExpFracAdjointControlled.qs",
+    # TODO(tfr): "Decompositions/ExpFracControlledAdjoint.qs",
+    "Decompositions/H.qs",
+    "Decompositions/HAdjoint.qs",
+    "Decompositions/HControlled.qs",
+    "Decompositions/HAdjointControlled.qs",
+    "Decompositions/HControlledAdjoint.qs",
+    "Decompositions/I.qs",
+    "Decompositions/IAdjoint.qs",
+    "Decompositions/IControlled.qs",
+    "Decompositions/IAdjointControlled.qs",
+    "Decompositions/IControlledAdjoint.qs",
+    "Decompositions/M.qs",
+    # TODO(tfr): not straightline input "Decompositions/MResetX.qs",
```

Comment:
Is not straightline input addressed in a subsequent PR?

Reply:
Yes, it is addressed in another PR.